README.md

```diff
@@ -25,6 +25,17 @@ $ export API_ENDPOINT='https://cells.dev.hubmapconsortium.org/api/'
 >>> [m for m in dir(client) if m.startswith('select_')]
 ['select_cells', 'select_clusters', 'select_datasets', 'select_genes', 'select_organs', 'select_proteins']
 
+>>> max_values_dict = {'rna':38.0, 'atac':0.31372055411338806, 'codex':40600.29881656805}
+
+>>> max_rna_value = client.get_max_value('rna')
+>>> assert max_rna_value == max_values_dict['rna']
+
+>>> max_atac_value = client.get_max_value('atac')
+>>> assert max_atac_value == max_values_dict['atac']
+
+>>> max_codex_value = client.get_max_value('codex')
+>>> assert max_codex_value == max_values_dict['codex']
+
```

Comment:
```suggestion
```

- We shouldn't hardcode these exact values in the tests: We don't want these tests to fail just because new data changed the bounds.
- I'd lean towards keeping the top-level readme short, and have tests that check bounds in a separate file... Do you think it's safe to say that rna < 100, atax < 1, and codex < 100000?

Reply:
Yeah, I'd had that same thought but couldn't think of a good way of testing without including values.  Now that I think about it, though, I'm not sure I'd feel confident in the bounds that you suggest, but I do think I'd feel confident that atac < rna < codex, if you think that's sufficiently strict.